### ISSUE TITLE
Expo Fastlane Build Failure - SPAlert

### ISSUE THREAD

**Issue by mvolonnino** [nandorojo/burnt#27]
Had these alerts hooked up and working which is why this is odd to me but getting fastlane build errors during my eas build phase for my expo app.

``` 
(ios/Pods/SPAlert/Sources/AlertKit/Views/AlertAppleMusic16View.swift:19:5)

  17 |         default: UIColor(red: 88 / 255, green: 87 / 255, blue: 88 / 255, alpha: 1)
  18 |         }
> 19 |     }
     |     ^ missing return in closure expected to return 'UIColor'
  20 |     
  21 |     fileprivate weak var viewForPresent: UIView?
  22 |     fileprivate var presentDismissDuration: TimeInterval = 0.2


❌  (ios/Pods/SPAlert/Sources/AlertKit/Views/AlertAppleMusic17View.swift:19:5)

  17 |         default: UIColor(red: 88 / 255, green: 87 / 255, blue: 88 / 255, alpha: 1)
  18 |         }
> 19 |     }
     |     ^ missing return in closure expected to return 'UIColor'
  20 |     
  21 |     fileprivate weak var viewForPresent: UIView?
  22 |     fileprivate var presentDismissDuration: TimeInterval = 0.2
  ```
  
  To make sure it was this libray, i had to remove `burnt`. Once i did was able to build again successfully.
  
    "burnt": "^0.11.7",
    "expo": "^49.0.0",
    "react-native": "0.72.5",
    
Followed the docs as well to get it working in Expo and like I said had it working and already made builds through EAS just fine.

I do have some code now that updates `Appearance` manually with `Appearance.setColorScheme()` -> I need this as my users can set their color mode to match the OS or be light & dark. If they select light or dark and the match os is toggled off, then native components did not update, so i added the Appearance.setColorScheme(light | dark | null) to match which works with things like `Alert` from RN, it also worked on development builds for your burnt alerts & toasts as well but been running into the error above trying to build new builds through EAS.

**Comment by mvolonnino** [comment]
https://reactnative-archive-august-2023.netlify.app/docs/appearance#setcolorscheme

**Comment by Taarna** [comment]
This is caused by SPAlert and it's already reported [here](https://github.com/sparrowcode/AlertKit/issues/62).

**Comment by mvolonnino** [comment]
@Taarna  - Ah okay cool! So it sounds like once thats fixed we should be good to add this back? Thanks for the reply!

**Comment by hiroingk** [comment]
I am also getting the SPAlert error.
I share the error message because I got a different message than the Issue author.

```
❌  (../../node_modules/burnt/ios/BurntModule.swift:13:64)

  11 |   case custom
  12 |   
> 13 |   func toSPAlertIconPreset(_ options: AlertOptions?) throws -> SPAlertIconPreset {
     |                                                                ^ cannot find type 'SPAlertIconPreset' in scope
  14 |     switch self {
  15 |       case .done:
  16 |         return .done


❌  (../../node_modules/burnt/ios/BurntModule.swift:39:29)

  37 |   case none
  38 |   
> 39 |   func toSPAlertHaptic() -> SPAlertHaptic {
     |                             ^ cannot find type 'SPAlertHaptic' in scope
  40 |     switch self {
  41 |       case .success:
  42 |         return .success


❌  (../../node_modules/burnt/ios/BurntModule.swift:245:18)

  243 |       }
  244 |       
> 245 |       let view = SPAlertView(
      |                  ^ cannot find 'SPAlertView' in scope
  246 |         title: options.title,
  247 |         message: options.message,
  248 |         preset: preset ?? .done)
```

**Comment by jordam1407** [comment]
Same error here using Tamagui Toast.... 
But in my case I'm trying to build on eas

Anyone found a fix for this?

![image](https://github.com/nandorojo/burnt/assets/91645575/e36e7080-e651-4693-8c2f-dbedd1e644e4)

**Comment by julien-lets** [comment]
I also encountered an EAS build issue (only for IOS). I first tried to build the app with the default settings (Xcode v14) and when doing that I saw similar error messages as the writer of this issue. Then I changed the image of the IOS builds to "latest" (Xcode v15) and encountered a different error message:

![image](https://github.com/nandorojo/burnt/assets/131444853/1def8307-e361-4d2c-b7cc-e0e256797154)

Also, I noticed that I am using burnt version 0.11.7, but I do not see this in the releases of the Github repo?

**Comment by nandorojo** [comment]
Can you try `burnt` version 0.12 (latest version) and see if that fixes it?

**Comment by jordam1407** [comment]
It's working, thanks!

**Comment by mvolonnino** [comment]
Yup, working for me as well, appreciate it! Already added it back in thats how NASTY these toasts are. Would love to see something close for Android as well!!

**Comment by nandorojo** [comment]
Fixes ship in under 24 hours — welcome to open source ⚡️

**Comment by julian-gargicevich** [comment]
@nandorojo  legend!
